Terasology/Potions
# Merge master to alpha, and issue #29

#29 When a potion is drank, and the effect is invalid, e.g.: has a typo in the asset file, an error message will be printed to the console.

![potioneffecterror](https://cloud.githubusercontent.com/assets/18668703/23839850/16e8bd0c-07a1-11e7-9ee5-8781a4b1d959.jpg)

***Steps to Test***
1. Open a potion asset, and modify the effect to something invalid. E.g.: open HarmPotion.prefab and change the effect to "HAAAARM"
2. Start up the game and create/open a world with the Potions module enabled
3. Add the potion from the console (give Potions:HarmPotion) and use it with right click
Expected Result: The effect will not happen; a red error message will appear in the console window.
